src/ampJson.js

```diff
@@ -1,7 +1,14 @@
-export default function ampJson(data, component) {
+export default function ampJson(data, component, attributes={}) {
   if (!data || !component) return '';
+
+  const attributesString = attributes ?
+    Object.entries(attributes).map(([key, val]) => (
+      `${key}=${val}`
+    )).join(' ').trim() :
+    undefined;
+
   return Object.entries(data).map(([key, value]) => (
-    `<${component} id="${key}">
+    `<${component} id="${key}" ${attributesString}>
```

Comment:
That still has the issue that `key` and `type` are being unsafely injected into HTML (to be fair, `component` is too, but we should really address all of them)